src/katello/repos.py

```diff
@@ -42,7 +42,7 @@ def upload_enabled_repos_report(report, force=False):
         return
     consumer_id = lookup_consumer_id()
     if consumer_id is None:
-        error_message('Cannot upload enabled repos report, is this client registered?')
+        error_message('Cannot upload enabled repos report, you need to be root to perform this command.')
```

Comment:
The original error message was correct in the case where the consumer_id is None (the host isn't registered), so I think that should stay the same. However, if the user is non-root, we should fail a bit earlier saying "hey, you need to be root!"